frontend/src/embedding-sdk/SdkProvider.tsx

```diff
@@ -0,0 +1,80 @@
+import type * as React from "react";
+import { Provider } from "react-redux";
+import { useEffect, useState } from "react";
+import { ThemeProvider } from "metabase/ui/components/theme/ThemeProvider";
+import { getStore } from "metabase/store";
+import reducers from "metabase/reducers-main";
+import registerVisualizations from "metabase/visualizations/register";
+import GlobalStyles from "metabase/styled-components/containers/GlobalStyles";
+import { setOptions } from "metabase/redux/embed";
+import api from "metabase/lib/api";
+import { reloadSettings } from "metabase/admin/settings/settings";
+import { refreshCurrentUser } from "metabase/redux/user";
+import { SdkEmotionCacheProvider } from "./SdkEmotionCacheProvider";
+import { EmbeddingContext } from "./context";
+import { SDK_CONTEXT_CLASS_NAME } from "./config";
+import "./styles.css";
+
+export const MetabaseProvider = ({
+  children,
+  apiUrl,
+  apiKey,
+  font,
+}: {
+  children: React.ReactNode;
+  apiUrl: string;
+  apiKey: string;
+  font?: string;
+}): JSX.Element => {
+  const store = getStore(reducers);
+
+  api.basename = apiUrl;
+  api.apiKey = apiKey;
+
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    Promise.all([
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
disabled for now since there's some typing issues with `refreshCurrentUser` which we can solve when we're polishing the PoC for presentation